client/signup/controller.js

```diff
@@ -113,22 +112,6 @@ export default {
 					const countryCode = geo.data.body.country_short;
 					if ( 'gutenberg' === abtest( 'newSiteGutenbergOnboarding', countryCode ) ) {
 						window.location.replace( window.location.origin + '/new' + window.location.search );
-					} else if (
-						-1 === context.pathname.indexOf( 'free' ) &&
-						-1 === context.pathname.indexOf( 'personal' ) &&
-						-1 === context.pathname.indexOf( 'premium' ) &&
-						-1 === context.pathname.indexOf( 'business' ) &&
-						-1 === context.pathname.indexOf( 'ecommerce' ) &&
-						'variantPasswordless' === abtest( 'passwordlessAfterPlans', countryCode )
-					) {
-						const stepName = getStepName( context.params );
-						const stepSectionName = getStepSectionName( context.params );
-						const urlWithoutLocale = getStepUrl(
-							'onboarding-passwordless',
-							stepName,
-							stepSectionName
-						);
-						window.location = urlWithoutLocale;
```

Comment:
You might want to include `removeWhiteBackground();` here? It's a bit hacky but:

- White background gets set by default for everywhere (so that there's no flash of blue before Gutenboarding flow)
- It gets removed for other than gutenboarding flows; `/start` signup flow has blue background.

Reply:
> I'm not familiar with your flow but you might want to include `removeWhiteBackground();` here? It's a bit hacky but:

That's a good catch and I think it reduces the time with the white background for a few milliseconds; until the is redirected redirected. 
Since we're now removing this flow, I don't think we'll need it, but it's definitely something we should keep in mind for future flow experiments.
Thank you! @simison